README.md

```diff
@@ -53,26 +53,30 @@ any new files in the repository.
 Make your own customizations
 ----------------------------
 
-Put your customizations in dotfiles appended with `.local`:
+Create a directory for your personal customizations: 
 
-* `~/.aliases.local`
-* `~/.git_template.local/*`
-* `~/.gitconfig.local`
-* `~/.gvimrc.local`
-* `~/.psqlrc.local` (we supply a blank `.psqlrc.local` to prevent `psql` from
+    mkdir ~/dotfiles-local
```

Comment:
I use `~/dotfiles-local` because the default [rcrc](https://github.com/thoughtbot/dotfiles/blob/master/rcrc#L2) file uses it. If you think some users will put their personal customizations in a different folder then we can perhaps include a note on how to use rcm with another folder